contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+### Contributing a new container
+
+You can contribute a new container in three steps:
+
+1. Create a new module with a class for the container at `testcontainers/[my fancy container].py` that implements the new functionality.
+2. Add a working usage example and short functionality description to the new class's docstring.
```

Comment:
Should we add the term 'doctest' here somehow to simplify the research on 'How to do that' for people who have not worked with doctests before? Maybe a reference to one of the existing?